### ISSUE TITLE
correct savepath

### ISSUE THREAD

**Issue by gschramm** [deepinv/deepinv#320]
https://github.com/deepinv/deepinv/blob/43475b6d538ede8492c81e00a94dd7d8c14663fc/deepinv/training/trainer.py#L298

I just tried running a few examples on my local Windows machine (for testing) and get the error:
```
OSError: [WinError 123] The filename, directory name, or volume label syntax is incorrect: 'ckpts\\denoising/24-10-11-13:49:37'
``

I think this is due to the line above where the should be:

```python
 str(self.save_path / f"{get_timestamp()}") if self.save_path else None 
```

otherwise "/" is used as a path separator on Windows.

**Comment by Andrewwango** [comment]
Hi @gschramm, thanks for raising this error. Looking at the filepath in the error you showed, it looks like there are colons in the directory name which are forbidden on Windows. However, `deepinv.utils.logger.get_timestamp` should use underscore instead of colons on Windows:

https://github.com/deepinv/deepinv/blob/43475b6d538ede8492c81e00a94dd7d8c14663fc/deepinv/utils/logger.py#L71

I think you are using an old version of the package, as this code was added in #183 and released in v0.2.1. Please can you `pip install git+https://github.com/deepinv/deepinv.git#egg=deepinv` and try again?

**Comment by gschramm** [comment]
indeed. I didn't use 0.2.1 (installed from conda-forge) where 0.2.1 isn't available yet.
